Doc/library/inspect.rst

```diff
@@ -905,10 +905,9 @@ Classes and functions
    are the names of the ``*`` and ``**`` arguments or ``None``.  *locals* is the
    locals dictionary of the given frame.
 
-   .. deprecated:: 3.5
-      Use :func:`signature` and
-      :ref:`Signature Object <inspect-signature-object>`, which provide a
-      better introspecting API for callables.
+   .. note::
+      This function was inadvertantly marked as deprecated in Python 3.5. Ignore
```

Comment:
"Ignore previous deprecation notice." is superfluous and can be deleted in my opinion.